qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -586,6 +587,24 @@ def _common_creation_steps(cls, md_template, obj_id):
             # Execute all the steps
             qdb.sql_connection.TRN.execute()
 
+    @classmethod
+    def metadata_headers(cls):
+        """Returns metadata headers available
+
+        Returns
+        -------
+        list
+            Alphabetical list of all metadata headers available
+        """
+        with qdb.sql_connection.TRN:
+            sql = """SELECT DISTINCT column_name
+                        FROM information_schema.columns
+                        WHERE table_name LIKE '{0}%' AND
+                              table_name != 'sample_template_filepath'
```

Comment:
should `prep_template_filepath` also be excluded here, now that this function is in the base class?

Having a test for the prep template case may help to detect other potential tables that should be ignored here.